src/main/java/com/powsybl/sld/server/SingleLineDiagramController.java

```diff
@@ -55,19 +55,15 @@ public class SingleLineDiagramController {
     static final String METADATA = "metadata";
     static final String ADDITIONAL_METADATA = "additionalMetadata";
 
+    // 3 maximum concurrent Network Area Diagram generations for the current configuration
+    private final ExecutorService executorService = Executors.newFixedThreadPool(3);
```

Comment:
Shouldn't we externalize the number of ThreadPool in a conf ?

Reply:
Done